src/Sylius/Bundle/AddressingBundle/Form/Type/CountryCodeChoiceType.php

```diff
@@ -61,10 +87,11 @@ private function getCountryCodes(array $countries)
 
         /* @var CountryInterface $country */
         foreach ($countries as $country) {
-            $countryCodes[$country->getCode()] = Intl::getRegionBundle()->getCountryName($country->getCode());
+            $countryName = Intl::getRegionBundle()->getCountryName($country->getCode());
+            $countryCodes[$countryName] = $country->getCode();
```

Comment:
It has started to fail since I've updated from 2.7.7 to 2.7.9 and `choice_as_values` is deprecated and defaults to `true` in Symfony >=3.0.